tests/test_topostats.py

```diff
@@ -0,0 +1,49 @@
+"""Test the loading of topostats (HDF5 format) files."""
+
+from pathlib import Path
+import pytest
+
+import numpy as np
+
+from AFMReader.topostats import load_topostats
+
+BASE_DIR = Path.cwd()
+RESOURCES = BASE_DIR / "tests" / "resources"
+
+
+@pytest.mark.parametrize(
+    ("file_name", "topostats_file_version", "image_shape", "pixel_to_nm_scaling", "data_keys", "image_sum"),
+    [
+        pytest.param(
+            "sample_0.topostats",
```

Comment:
Perhaps pedantic but to make it easier to read/understand what's what in the `tests/resources/` directory we could use a nomenclature that reflects the version.

```suggestion
            "sample_0_1.topostats",
```

...would indicate the version `0.1` (would also require renaming `sample_0.topostats` > `sample_0_1.topostats`).